pkg/handlers/template_test.go

```diff
@@ -321,6 +321,22 @@ func TestTemplateHandlerValidation(t *testing.T) {
 				"CLUSTERNAME":            "testcluster",
 			},
 		},
+		{
```

Comment:
as this test grows, breaking it into multiple files would make sense to organize the cases by template

Reply:
Agreed! We can pull out the testing function for resuse